src/executables/sysrepocfg.c

```diff
@@ -930,6 +931,178 @@ srcfg_import_datastore(struct ly_ctx *ly_ctx, int fd_in, md_module_t *module, sr
     return rc;
 }
 
+/**
```

Comment:
Now both srcfg_merge_datastore() and srcfg_import_operation() have a new parameter (boot merge) to select import (merge=flase) or merge operation (merge=true)